SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md

```diff
@@ -31,8 +31,8 @@ Before porting your project:
     For example:
     `\StarterProject\Game\StarterProject.uproject`
     
-    > This step is essential as the `spatial` folder must be located in the directory above your `<GameRoot>`. This is so the GDK scripts work correctly with Unreal.
-1. Your game's project needs some extra files and folders to run with the GDK; you can copy these from the StarterProject repository that you cloned earlier in the [Before you start](#before-you-start) section.
+    > This step is essential as the `spatial` folder must be located in the directory above your `<GameRoot>`. This is so that the GDK scripts work correctly with Unreal.
+1. Your game's project needs some extra files and folders to run with the GDK; you can copy these from the `StarterProject` [repository](https://github.com/spatialos/UnrealGDKStarterProject). 
```

Comment:
We told people above to not clone the Starter repo, and now we were telling them they shoulda.

Reply:
Oh, then tell them to clone that baby.

Reply:
I've no idea how that happened honestly. I've just looked at my branch and this wasn't in my PR.